api/server.js

```diff
@@ -27,7 +27,7 @@ require('./routes/loginUser')(app);
 require('./routes/registerUser')(app);
 require('./routes/findUser')(app);
 // require('./routes/deleteUser')(app);
```

Comment:
Is this just here commented out because you're doing it in a future PR?

Reply:
It was commented out when I was working on it, I didnt write that comment

Reply:
Ohhh weird haha nvm